imi/src/main/java/org/motechproject/nms/imi/service/impl/CdrFileServiceImpl.java

```diff
@@ -90,6 +167,8 @@ private void parseDetailRecord(CallDetailRecordDto cdr, Map<String, CallSummaryR
                 record.setSecondsPlayed(cdr.getMsgPlayDuration());
             }
 
+            //todo: maybe verify the attemptCount provided by IMI and see if there are holes?
+
             /*
                Update the final status
 
```

Comment:
See future checkin that'll say something like "cdr processing, take four: now sorting the incoming file"...